### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by drivemax26**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.10.4
```
Comment (Abnormaltype) [inline Dockerfile:1]: The image of the service should be as thin as possible;

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+import os
+
+key = os.environ.get('API_KEY')
+
+
 def get_weather():
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    res = requests.get(f"http://api.weatherapi.com/v1/current.json?key={key}&q=Paris&aqi=no").json()
```
Comment (Abnormaltype) [inline app/main.py:9]: create a constant for this URL

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+key = os.environ.get('API_KEY')
+
+city = "Paris"
+
+
 def get_weather():
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    url = "http://api.weatherapi.com/v1/current.json"
```
Comment (Abnormaltype) [inline app/main.py:11]: make a constant, please

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+    payload = {"key": key, "q": city}
```
Comment (Abnormaltype) [inline app/main.py:12]:
```suggestion
    payload = {"key": key, "city": city}
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 4 above)
Comment (Abnormaltype) [reply app/main.py:13]: and this)

File: app/main.py [hunk 5]
```diff
@@ -1,6 +1,26 @@
+import requests
+import os
+
+key = os.environ.get('API_KEY')
+
+city = "Paris"
```
Comment (Abnormaltype) [inline app/main.py:6]:
```suggestion
CITY = "Paris"
```

File: app/main.py [hunk 6]
```diff
@@ -1,6 +1,26 @@
+import requests
+import os
+
+key = os.environ.get('API_KEY')
```
Comment (Abnormaltype) [inline app/main.py:4]:
```suggestion
KEY = os.environ.get('API_KEY')
```
OR 
```suggestion
API_KEY = os.environ.get('API_KEY')
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 7]
```diff
@@ -1,6 +1,24 @@
+import requests
+import os
+
+API_KEY = os.environ.get('API_KEY')
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+PAYLOAD = {"key": API_KEY, "q": CITY}
```
Comment (Abnormaltype) [inline app/main.py:7]: why the key is q?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by drivemax26** [comment]
"q" is API parameter, if I change to CITY , API returns an error

File: app/main.py (hunk 7 above)
Comment (drivemax26) [reply app/main.py:7]: "q" is API parameter, if I change to CITY , API returns an error

**Review by Abnormaltype — APPROVED** [review]
Well done!

**Pull request closed without merging**